charts/op-scim-bridge/values.yaml

```diff
@@ -103,11 +103,11 @@ scim:
   # resource sets the requests and/or limits for the SCIM bridge pod
   resources:
     requests:
-      cpu: 125m
-      memory: 256M
-    limits:
       cpu: 250m
       memory: 512M
+    limits:
+      cpu: 500m
+      memory: 1024M
```

Comment:
Are these values correct?

It looks like we are increasing from 125m (which I believe we have everywhere else) to 250m for the SCIM Bridge pod.

Reply:
Fixed in the latest commit 🤦🏼